pkg/scheduler/apis/config/v1beta1/defaults_test.go

```diff
@@ -409,13 +409,23 @@ func TestPluginArgsDefaults(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:    "PodTopologySpreadArgs empty array, DefaultPodTopologySpread feature enabled",
+			feature: features.DefaultPodTopologySpread,
+			in: &v1beta1.PodTopologySpreadArgs{
+				DefaultConstraints: []v1.TopologySpreadConstraint{},
+			},
+			want: &v1beta1.PodTopologySpreadArgs{
+				DefaultConstraints: []v1.TopologySpreadConstraint{},
+			},
+		},
```

Comment:
There can add a test for the DefaultPodTopologySpread feature disabled.

A question, why we add those default test to encode/decode tests at scheme_test.go, seems the tests in defaults_test.go should be enough.

Reply:
> There can add a test for the DefaultPodTopologySpread feature disabled.

That's the default, so it's already tested above.

> A question, why we add those default test to encode/decode tests at scheme_test.go, seems the tests in defaults_test.go should be enough.

Because tests in scheme_test.go are more e2e-like, ensuring that we didn't mistype function names and that our assumptions of what the api-machinery does align with what we expect (nil vs empty array).